lib/libtlspool.c

```diff
@@ -569,15 +345,9 @@ static void *master_thread (void *path) {
 					// TLS Pool is waiting for a callback;
 					// Send it an ERROR message instead.
 					cmd.pio_cmd = PIOC_ERROR_V2;
-					cmd.pio_data.pioc_error.tlserrno = E_TLSPOOL_CLIENT_REFUSES_CALLBACK;
-					strncpy (cmd.pio_data.pioc_error.message,
-							"TLS Pool client will not partake in callback",
-							sizeof (cmd.pio_data.pioc_error.message));
-#ifdef WINDOWS_PORT
-					np_send_command (&cmd);
-#else
-					sendmsg (poolfd, &mh, MSG_NOSIGNAL);
-#endif
+					cmd.pio_data.pioc_error.tlserrno = EPIPE;
+					strncpy (cmd.pio_data.pioc_error.message, "Client prematurely left TLS Pool negotiations", sizeof (cmd.pio_data.pioc_error.message));
```

Comment:
ook hier is EPIPE het oude systeem; als we distributed gaan kunnen we errno niet langer handhaven (al geldt dat niet voor libtlspool) want die codes liggen niet vast en zijn bovendien van een ander soort systeem.  Kun je hiervan maken E_TLSPOOL_CLIENT_DISCONNECT met als message "TLS Pool connection closed"